packages/atlas/src/providers/videoWorkspace/hooks.ts

```diff
@@ -34,6 +34,19 @@ export const useVideoWorkspaceData = () => {
     onError: (error) => SentryLogger.error('Failed to fetch video', 'useVideoWorkspaceData', error),
   })
 
+  const subtitlesArray = useMemo(
+    () =>
+      video?.subtitles.map(
+        (s) =>
+          ({
+            languageIso: s.language.iso,
+            type: s.type === 'closed-captions' ? 'closed-captions' : 'subtitles',
```

Comment:
hmm, can't we just use `s.type` here?

Reply:
Not really, because `s.type` is of type `string`, I need to create additional Typescript guards so we limit the type to `'subtitles' | 'closed-captions'`

Reply:
What about adding `... as const`? Shouldn't do the job?

Reply:
nvm, you're right. This will also not work. We're getting this from graphql.